plugins/aws-costs/aws-costs.sh

```diff
@@ -62,12 +67,18 @@ while [ ! $# -eq 0 ]; do
     shift
     shift
 done
-if [[ "$tagkey" = "" || "$tagvalue" = "" || "$database" = "" || "$table" = "" || "$catalog" = "" || "$workgroup" = "" ]]; then
+if [[ "$days" = "" && "$AWS_COSTS_DAYS" != "" ]]; then
+  days=$AWS_COSTS_DAYS
+fi
+if [[ "$days" = "" ]]; then
+  days='5'
+fi
+if [[ "$tagkey" = "" || "$tagvalue" = "" || "$database" = "" || "$table" = "" || "$catalog" = "" || "$workgroup" = "" || "$days" = "" ]]; then
   usage
   exit 1
 fi
 
-initial_date_time=$(date -u -d '5 day ago' +"%Y-%m-%d %H:00:00.000")
+initial_date_time=$(date -u -d  $days+' day ago' +"%Y-%m-%d %H:00:00.000")
```

Comment:
It seems like this will yield `5' day ago'` instead of `'5 day ago'`

Reply:
I think it's fine:
james@DESKTOP-A3283C9:~$ export days=5
james@DESKTOP-A3283C9:~$ initial_date_time=$(date -u -d  $days+' day ago' +"%Y-%m-%d %H:00:00.000")
james@DESKTOP-A3283C9:~$ echo $initial_date_time
2023-02-18 13:00:00.000
james@DESKTOP-A3283C9:~$

Reply:
The string calculates the relative date via the date command - it returns output like: 2023-02-18 13:00:00.000